styles.css

```diff
@@ -728,3 +754,666 @@ a {
   border: 1px solid #f55800;
   color: #f55800;
 }
+
+/* Desktop version */
+
+@media screen and (min-width: 768px) {
+  * {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    height: auto;
+
+    /* border: solid yellow 0.5px; */
+  }
+
+  body {
+    max-width: 90rem;
+  }
+
+  a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .nav_bar {
+    width: 1440px;
+    height: 72px;
+    left: 0;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 24px;
+    background: #3c3a39;
+    overflow: hidden;
+  }
+
+  .nav_menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 45px;
+    padding-bottom: 10px;
+    margin-right: 80px;
+    list-style: none;
+  }
+
+  .nav_branding {
+    font-size: 20px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 90px;
+    margin-bottom: 10px;
+  }
+
+  .nav_link {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-right: 10px;
+    transition: 0.7s ease;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+  }
+
+  .email_icon {
+    margin-right: 20px;
+  }
+
+  .nav_link:hover {
+    color: #f55800;
+  }
+
+  .hamburger_icon {
+    display: none;
+  }
+
+  .nav_email_icon {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #fff;
+  }
+
+  .hr-line_break3 {
+    display: none;
+  }
+
+  .mobile_nav_container {
+    display: none;
+  }
+
+  .main_section {
+    background-color: #1c1a19;
+    background-image: url('./image folder/Illustration-header.png');
+    background-position: right bottom;
+    background-repeat: no-repeat;
+    background-size: auto;
+    height: 800px;
+    padding: 0 24px;
+    color: white;
+  }
+
+  .main_section_div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 12rem 0.5rem 12rem 12rem;
+    gap: 24px;
+    width: 35.38rem;
+    height: 46.19rem;
+    left: 143px;
+    top: 72px;
```

Comment:
actually  to create a responsive design we should used flex box and grid layouts to align our blocs rem can be used to text size